src/main/webapp/GeppettoConfiguration.json

```diff
@@ -1,9 +1,10 @@
 {
     "_README" : "http://docs.geppetto.org/en/latest/build.html",
-    "contextPath": "org.geppetto.frontend",
+    "contextPath": "/org.geppetto.frontend",
```

Comment:
@jrmartin I'm testing locally but I had to remove the / for anything to work. Was there a reason to add it?